sound/soc/sof/sof-priv.h

```diff
@@ -301,6 +301,7 @@ struct snd_sof_pcm_stream {
 	struct sof_ipc_stream_posn posn;
 	struct snd_pcm_substream *substream;
 	struct work_struct period_elapsed_work;
+	bool d0i3_compatible; /* pcm can run at low power like d0i3 */
```

Comment:
@keyonjie the comment is more meaningful when you say something like "DSP can be in D0I3 when this pcm is open"

Reply:
yes, please don't use 'run' since it's not a state or wording that we use. And as Ranjani mentioned the DSP is in low-power, not the PCM.